grammar/argextend.mlp

```diff
@@ -46,17 +46,17 @@ let make_act loc act pil =
   make (List.rev pil)
 
 let make_prod_item = function
-  | ExtTerminal s -> <:expr< Grammar_API.Extend.Atoken (Grammar_API.CLexer.terminal $mlexpr_of_string s$) >>
+  | ExtTerminal s -> <:expr< API.Extend.Atoken (API.CLexer.terminal $mlexpr_of_string s$) >>
```

Comment:
Emilio are you implicitly saying that ARGUMENT EXTEND is used in both main-coq code and plugin code?  If so I understand your remark, if not then I don't see why making generated code more "context dependent" could be a plus.